src/components/EditableCard.tsx

```diff
@@ -55,50 +64,66 @@ class EditableCard extends Component<IEditableCardProps> {
   }
 
   private get deleteButton () {
-    const { isGuarded, title, onDelete, isLoading } = this.props
-      , classNameSuffix = this.props.classNameSuffix || kebabCase(title);
-
+    const {
+      classNameSuffix,
+      disableDeleteTooltip,
+      disableDelete,
+      isGuarded,
+      isLoading,
+      onDelete,
+      title,
+    } = this.props,
+    className = getBtnClassName('delete', classNameSuffix, title);
     if (!onDelete) { return; }
 
     return (
-      <GuardedButton
-        className={`btn-delete btn-delete-${classNameSuffix}`}
-        confirm={true}
-        disabled={isLoading || this.isDeleting.isTrue}
-        icon='delete'
-        isGuarded={isGuarded}
-        onClick={this.handleDelete}
-        size='small'
-        type='danger'
-      >
-        Delete
-      </GuardedButton>
+      <Tooltip title={disableDelete ? disableDeleteTooltip : ''}>
+        <span>
+          <GuardedButton
+            className={className}
+            confirm={true}
+            disabled={disableDelete || isLoading || this.isDeleting.isTrue}
+            icon='delete'
+            isGuarded={isGuarded}
+            onClick={this.handleDelete}
+            size='small'
+            type='danger'
+          >
+            Delete
+          </GuardedButton>
+        </span>
+      </Tooltip>
     );
   }
 
   private get editButton () {
-    const { isLoading, title, isGuarded } = this.props
+    const { isLoading, title, isGuarded, disableEdit, disableEditTooltip } = this.props
       , classNameSuffix = this.props.classNameSuffix || kebabCase(title);
 
     return (
-      <GuardedButton
-        className={`btn-edit btn-edit-${classNameSuffix}`}
-        disabled={isLoading || this.isEditing.isTrue || this.isDeleting.isTrue}
-        icon='edit'
-        isGuarded={isGuarded}
-        onClick={this.isEditing.setTrue}
-        size='small'
-        type='primary'
-      >
-        Edit
-      </GuardedButton>
+      <Tooltip title={disableEdit ? disableEditTooltip : ''}>
+        <span>
+          <GuardedButton
+            className={`btn-edit btn-edit-${classNameSuffix}`}
+            disabled={isLoading || this.isEditing.isTrue || this.isDeleting.isTrue || disableEdit}
+            icon='edit'
+            isGuarded={isGuarded}
+            onClick={this.isEditing.setTrue}
+            size='small'
+            type='primary'
+          >
+            Edit
+          </GuardedButton>
+        </span>
+      </Tooltip>
     );
   }
 
   private buttons () {
     return (
       <ButtonToolbar noSpacing>
         {this.deleteButton}
+        <div style={{ display: 'inline-block', width: '10px' }}></div>
```

Comment:
Can we try using an `antd` `Space` for this? i.e.
```
import { Space } from 'antd';

function MyComponent() {
  return (
    <Space size={10}>
      {this.deleteButton}
      {this.editButton}
    </Space>
  );
}
```

Reply:
`Space` component isnt supported in antd3

Reply:
hmm could we try making a parent `div` with `display: flex`, `gap: 10px`, and `justify-content: space-between` and see if that works? Would like to try to avoid the empty `<div>` if possible, though if that doesn't work I'm OK with this.

it also looks like we don't have much custom styling in this library since we mostly just use the built-in `antd` functions, so I think inline styling is OK as a workaround for now since this is just a branch. ideally on the `main` branch we will eventually upgrade to `antd4` and use `<Space>` for things like this.

Reply:
actually deleting the `span`s made this not needed. im seeing the tooltips when the buttons are disabled so not sure what you saw in the previous PR? 🤔

Reply:
maybe antd3 vs 4 difference?